sdk/python/setup.py

```diff
@@ -257,7 +322,13 @@ def run(self):
     ],
     entry_points={"console_scripts": ["feast=feast.cli:cli"]},
     use_scm_version=use_scm_version,
-    setup_requires=["setuptools_scm", "grpcio", "grpcio-tools==1.34.0", "mypy-protobuf==3.1.0", "sphinx!=4.0.0"],
+    setup_requires=[
```

Comment:
Why did so much formatting in this file change?

Reply:
because I manually ran format and lint commands on it